tests/unit/s2n_psk_offered_test.c

```diff
@@ -198,7 +198,8 @@ int main(int argc, char **argv)
 
     /* Test s2n_offered_psk_new */
     {
-        struct s2n_offered_psk zeroed_psk = { 0 };
+        struct s2n_offered_psk zeroed_psk;
```

Comment:
Nit: you should still probably do "= { 0 }", just so nobody thinks you're not initializing a variable :)